health-services/project/src/main/resources/application.properties

```diff
@@ -49,17 +49,25 @@ kafka.producer.config.linger_ms_config=1
 kafka.producer.config.buffer_memory_config=33554432
 
 # IDGEN CONFIG
-egov.idgen.host=https://dev.digit.org/
+# egov.idgen.host=https://dev.digit.org/
+egov.idgen.host=http://localhost:8088/
```

Comment:
as a practice should our properties files point to local or dev env?

Reply:
Updated